workspaces/src/network/server.rs

```diff
@@ -37,9 +38,14 @@ impl SandboxServer {
 
         // Remove dir if it already exists:
         let _ = std::fs::remove_dir_all(&home_dir);
-        sandbox::init(&home_dir)?.wait()?;
+        sandbox::init(&home_dir)
+            .map_err(|e| Error::SandboxUnknownError(e.to_string()))?
+            .wait()
+            .map_err(Error::IoError)?;
```

Comment:
Have you considered a separate error type for sandbox initialization issues?

Reply:
good point, added it